### DISCUSSION TITLE
Hyper fruits on red combo colors are hard to see in osu!catch with argon skin

### DISCUSSION THREAD

**Discussion by Vekter000** [ppy/osu#22592] [Ideas]
I am having a hard time discerning hyper fruits on red combo colors with the argon skin. This beatmap is a good example: https://osu.ppy.sh/beatmapsets/350076#fruits/771881

Here are what two hyper fruits on red combo colors look like on a red background in argon: 
![ByeU1Ew](https://user-images.githubusercontent.com/30701113/217973112-bd4e9e31-07c9-4b8b-86b7-5dd07a5c61e1.png)

and in triangles: 
![SvEnOYV](https://user-images.githubusercontent.com/30701113/217973135-356c9436-7bf2-4bec-a4f8-cd095fea5b03.png)

Could argon be changed to also give the fruit's inner part a red highlight? Or change the middle dot color to red?
Hyper dashes can often be implied, but as a newer player I find myself reverting back to the triangles skin for visibility.

**Comment by bdach** [comment]
This seems pretty valid. cc @arflyte (surely it will work some time?)
